src/utils.ts

```diff
@@ -173,3 +174,33 @@ export const deepEqual = (obj1: any, obj2: any): boolean => {
 
   return true;
 };
+
+export const getColumnNameByCase = (columnName: string) => {
+  const showColumnNamesInLowercase = workspace
+    .getConfiguration("dbt")
+    .get<boolean>("showColumnNamesInLowercase", true);
+  return showColumnNamesInLowercase ? columnName.toLowerCase() : columnName;
+};
+
+export const isColumnNameEqual = (
+  columnNameA: string | undefined,
+  columnNameB: string | undefined,
```

Comment:
Maybe it is better to call them by what is the source of the column name. if A is coming from yaml and B from database.

that way it will be easier later on if we need to process the one other than the other.